.github/workflows/main.yml

```diff
@@ -1,12 +1,11 @@
-name: Comment Build Stats
-
+name: Main
 on:
     pull_request:
-        branches-ignore:
-            - '**'
+        branches: ['master']
 
 jobs:
-    build_and_comment:
+    generate_stats:
+        name: Generate Build Stats
```

Comment:
I would be a bit more specific with the name here. Right now, seeing the failure does not immediately point to the problem:

<img width="699" alt="Screen Shot 2020-04-16 at 09 47 20" src="https://user-images.githubusercontent.com/905006/79429031-4ae59200-7fc7-11ea-9dc0-e7c49ffe0763.png">

If we're simply using this job to `checkSize`, I would go with something like "Bundle Size" that clearly hints what's being tested there...